index.js

```diff
@@ -16,30 +23,46 @@ function moveToMapPosition(referenceMap, mapToMove) {
 // - could cause an infinite loop
 // - prematurely halts prolonged movements like
 //   double-click zooming, box-zooming, and flying
-function syncMaps(a, b) {
-  on();
+function syncMaps () {
+  var maps;
+  var argLen = arguments.length;
```

Comment:
@tmcw The idea was to be backward compatible. I'd prefer an array as well, but since the original plugin expects discreet arguments I thought of keeping it so.